src/auth-service/bin/index.js

```diff
@@ -1,12 +1,27 @@
+require("module-alias/register");
+const dotenv = require("dotenv");
+dotenv.config();
+require("app-module-path").addPath(__dirname);
 const kafkaConsumer = require("./kafka-consumer");
-const sendEmail = require("@utils/mailer");
 const createServer = require("./server");
+const log4js = require("log4js");
+const constants = require("@config/constants");
+
```

Comment:
We now have a new entry file for the application. the app.js has been removed. @BenjaminSsempala